h5py/h5d.pyx

```diff
@@ -395,3 +395,38 @@ cdef class DatasetID(ObjectID):
             """
             H5Drefresh(self.id)
 
+
+    IF HDF5_VERSION >= (1, 8, 11):
+
+        def write_direct_chunk(self, offsets, bytes data, H5Z_filter_t filter_mask=H5Z_FILTER_NONE, PropID dxpl=None):
+            """ no return
```

Comment:
Here's a good example from the existing low-level interface:

https://github.com/h5py/h5py/blob/master/h5py/h5d.pyx#L84